modules/account-lib/src/coin/stx/keyPair.ts

```diff
@@ -76,9 +76,18 @@ export class KeyPair extends Secp256k1ExtendedKeyPair {
    * @returns {DefaultKeys} The keys in the protocol default key format
    */
   getKeys(compressed = false): DefaultKeys {
+    let prv;
```

Comment:
nit: can we reformat to 

`let prv;`
`prv = this.keyPair.d ? this.keyPair.d.toBuffer(32).toString('hex') : undefined`
`prv = prv && compressed ? prv + '01' : prv`

Reply:
I've pushed a change for this, commit 387f2c3ed87789b8256ba802b15c56e5e9b947c7.